tests/test-sources/modules/performance/byte-compile-lua.nix

```diff
@@ -1,5 +1,7 @@
-{ pkgs, ... }:
+{ lib, pkgs, ... }:
```

Comment:
If this is just for the `pluginStubs`, we can revert the change if using `callPackages`.

Reply:
I noticed a usage of `pkgs.lib.genAttrs` later on, so I'll leave a lib argument here. When I wrote this initially, lib wasn't passed to tests, but now it is.

Reply:
I want an opinion though: is it better to have a lib, pkgs on every test where it is used, or one top level is enough? Looking more closely, I noticed, that after this change I started to use a top-level lib in one of the tests. It worked and I didn't notice that. Should I care about it? So I should either remove a lib from test functions (and use a top-level one), or add a lib to every test where it's used. Which is better?

Reply:
In practice, there should be very little difference.

The underlying `lib` and `pkgs` passed to test files _should_ be the same `lib` and `pkgs` available as module args within the test-case configurations.

However looking at the code, that isn't the case(!)

[`fetchTests`](https://github.com/nix-community/nixvim/blob/69266437beaac970568d12a46766706c72f361b3/tests/main.nix#L12) uses the `pkgs` instantiated by flake-parts, while the test-case configurations uses a [`pkgsForTest`](https://github.com/nix-community/nixvim/blob/69266437beaac970568d12a46766706c72f361b3/tests/main.nix#L16-L19) instance.

Still, in practice it usually won't matter, other than the small performance hit of having two pkgs instances, which likely can't be avoided _anyway_.